blockchain/My-Wallet-V3-iOS
# fix(Homebrew-Exchange): IOS-1382 remove order ID placeholder and display order ID in Exchange Locked screen

## Objective

Show order ID in the Exchange Locked screen.

## Description

- Added mutable string property `orderIdentifier` to `OrderTransaction`.
- Set `orderIdentifier` from the `OrderResult`'s `id` returned from the trades endpoint.
- Set the `orderIdentifier` on the `OrderTransaction` just before calling the `.sentTransaction` event.
- Added order ID to the Exchange Locked screen as a new cell.

## How to Test

Send an Exchange Order and see order ID displayed. You should be able to tap and copy the order ID to the clipboard.

## Screenshot/Design assessment
Not styled the same way as in the spec (in a single line).
5S:
![5s](https://user-images.githubusercontent.com/10579422/46378143-b9266600-c668-11e8-8781-568f5a74d17e.png)
6:
![6](https://user-images.githubusercontent.com/10579422/46378158-bfb4dd80-c668-11e8-9608-7a81ef65ef16.png)
6 plus:
![6 plus](https://user-images.githubusercontent.com/10579422/46378206-dd824280-c668-11e8-881a-928e18782172.png)

## Merge Checklist
